variables.tf

```diff
@@ -68,7 +68,11 @@ variable "managed_identity_object_id" {
 }
 
 variable "sku" {
-  type = "string"
-  default = "Standard"
+  type        = "string"
+  default     = "Standard"
   description = "SKU type(Basic, Standard and Premium)"
 }
+
+variable "mgmt_subscription_id" {}
+variable "aks_subscription_id" {}
+variable "aks_preview_subscription_id" {}
```

Comment:
this value needs to be set not really sure how you plan on doing this though, I guess default it to the DCD-CFTAPPS-DEV subscription ID and then override it in sandbox to be DCD-CFTAPPS-SBOX id?

Reply:
yes as per original tfvars, was just validating in sbox